app/models/article.rb

```diff
@@ -83,13 +83,48 @@ def link
 
   def year
     if !self.issue.year.blank?
-       self.issue.year
+      self.issue.year
     else 
-       "[BRAK ROKU WYDANIA]"
+      "[BRAK ROKU WYDANIA]"
     end
   end
 
   def to_param
     [id, title.parameterize].join("-")
   end
+  
+  def generate_certificate
```

Comment:
Proszę przenieść to do osobnego pliku, np. `app/models/certificate.rb`. W przeciwnym razie nie zachowa Pani zasady SRP https://en.wikipedia.org/wiki/Single_responsibility_principle